src/index.css

```diff
@@ -74,3 +74,7 @@ body {
 .noBorder {
   border: none;
 }
+
+.show-on-hover:hover > .show-on-hover {
+  display: block !important;
+}
```

Comment:
question: It seems hacky way to achieve this effect. Maybe would be better to use js to control when element has mouse on it?